lms/djangoapps/shoppingcart/models.py

```diff
@@ -1746,6 +1746,8 @@ def purchased_callback(self):
         self.course_enrollment.activate()
 
     def additional_instruction_text(self):
+        if not self.course_enrollment.is_verified_enrollment():
+            return ""
```

Comment:
My understanding is that the refund policy is the same as for verified courses.  The text on lines 1752 will need to be updated so it doesn't say "Verified Certificate".

@srpearce How does this sound for the refund reminder (for both verified / non-verified cases):
```
You have up to two weeks into the course to unenroll and receive a full refund.
To receive your refund, contact {billing_email}.
Please include your order number in your email.
Please do NOT include your credit card information.
```